ci/scripts/check_dv_sw_images.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+"""Checks for Bazel targets in sim DV cfg files.
+"""
+
+import sys
+import hjson
+from pathlib import Path
+
+from lib.bazel_query import BazelQueryRunner
+
+# List of fake targets added to please the linker:
+FAKE_TARGETS = [
+    "@manufacturer_test_hooks//:example_test",
+]
+
+
+def check_sw_image(name, sw_image, valid_sw_images):
+    # The format is target:type[:flags]...
+    # where the target is a bazel name so it also contains a ':'
+    # and the type indicate whether it's a flash image, otp image, and so on.
+    try:
+        bazel_module, bazel_label, other = sw_image.split(':', 2)
+    except ValueError:
+        print(f"- {name}: invalid sw_image '{sw_image}'")
```

Comment:
I think I'd suggest tweaking `check_sw_image` so that it returns a (hopefully empty) list of problems. The `print` lines here could then be turned into `acc += "my message"` and the true/false checks would be replaced with checking whether a list of failures was empty.